File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/test/ParamsItem.java
package com.github.kotvertolet.youtubejextractor.models.test;

import com.google.gson.annotations.SerializedName;

public class ParamsItem{

	@SerializedName("value")
	private String value;

	@SerializedName("key")
	private String key;

	public void setValue(String value){
		this.value = value;
	}

	public String getValue(){
		return value;
	}

	public void setKey(String key){
		this.key = key;
	}

	public String getKey(){
		return key;
	}
}

File: youtubejextractor/src/main/java/com/github/kotvertolet/youtubejextractor/models/newModels/ParamsItem.kt
package com.github.kotvertolet.youtubejextractor.models.newModels

import android.os.Parcelable
import com.google.gson.annotations.SerializedName
import kotlinx.android.parcel.Parcelize
import java.io.Serializable

@Parcelize
class ParamsItem : Parcelable, Serializable {
    @SerializedName("value")
    var value: String? = null

    @SerializedName("key")
    var key: String? = null
    override fun toString(): String {
        return "ParamsItem{" +
                "value = '" + value + '\'' +
                ",key = '" + key + '\'' +
                "}"
    }
}